src/snap-points.ts

```diff
@@ -285,12 +285,21 @@ export const createSnapshotForNetwork = async (network: Network) => {
         totalPoints,
         positionsAmount: eventsObject[curr].active.length,
         last24HoursPoints: 0,
+        position: 0,
```

Comment:
Its confusing if it is position in leaderboard or user positions. I'll rename it to a `rank` or something similiar